### ISSUE TITLE
Release zip

### ISSUE THREAD

**Issue by boomboo** [Tomslack/ManbabyDungeonTools#27]
The release zip should probably not contain the python dir or the .git dir. OopsIUpdated.txt is questionable I guess.

**Comment by jarxjarx** [comment]
Agreed. Should probably sort a new release considering #26 also makes route-sharing with other versions compatible again, so not everyone needs to have the addon from the same source.

**Comment by srenauld** [comment]
Github workflow coming up in a bit.

**Comment by srenauld** [comment]
@boomboo Okay, it is done. We have automated releases. You can find the latest (working) one here: https://github.com/Tomslack/ManbabyDungeonTools/releases/tag/v1.1.2 . `python`, all the git stuff and all dotfiles should be stripped.

As a bonus, we gained the ability to not have to keep the TOC version in sync!

Close when you've verified the release

**Comment by srenauld** [comment]
Oh, and @jarxjarx , the route sharing should be in it.

**Comment by boomboo** [comment]
Looks good, apart from OopsIUpdated.txt unless you specifically decided to keep it.
